controller/createVNubanController.js

```diff
@@ -37,7 +34,7 @@ async function sendVNubanFailureEmail(data, err) {
   try {
     const emailBody = {
       toEmails: failureEmailReceivers,
-      subject: 'vNuban Creation Failure',
+      subject: (appConfig.ENVIRONMENT == 'production' ? 'Production' : 'Staging') + ' - vNuban Creation Failure',
```

Comment:
For production env, keep subject as "Notification of vNuban creation failure"
For staging, pre fix the subject as "Test email: Notification of vNuban creation failure"